### PR TITLE
Bug 1666573: Add CLI download link

### PR DESCRIPTION
**Pull request by spadgett**
https://bugzilla.redhat.com/show_bug.cgi?id=1666573

/assign @rhamilto

### CODE REVIEW

**Comment by spadgett** [comment]
updated

/skip

File: frontend/public/const.js [hunk 1]
```diff
@@ -36,3 +36,5 @@ export const RH_OPERATOR_SUPPORT_POLICY_LINK = 'https://access.redhat.com/third-
 
 // Package manifests for the OperatorHub use this label.
 export const OPERATOR_HUB_LABEL = 'openshift-marketplace';
+
+export const CLI_DOWNLOAD_LINK = 'https://mirror.openshift.com/pub/openshift-v4/clients/4.1';
```
Comment (dtaylor113) [inline frontend/public/const.js]: LGTM, can we use a 'latest' for instead of hardcoding to 4.1?  We'll have to change this after next release right?

Comment (spadgett) [reply frontend/public/const.js]: We don't want a 4.1 cluster linking to a 4.2 (or 4.3 or 4.4...) CLI download. We're looking at providing a download in-cluster that would always have the right version, so hopefully this is a stop-gap solution.

**Comment by spadgett** [comment]
/hold

Looks like the link might be changing: https://bugzilla.redhat.com/show_bug.cgi?id=1666573#c17

**Comment by spadgett** [comment]
Updated the link to https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.1

**Pull request merged**